test/fixtures/hdfs-fixture/build.gradle

```diff
@@ -86,6 +130,17 @@ tasks.withType(ShadowJar).configureEach {
   }
 }
 
+tasks.register('showHdfs3Jar', Copy) {
```

Comment:
I don't see what these tasks are used for? Did you add them just to debug what was in these configurations?

Reply:
cleaning up. only used for some debugging

Reply:
removed